docs/algorithms/mce_irl.rst

```diff
@@ -13,6 +13,8 @@ Detailed example notebook: :doc:`../tutorials/6_train_mce`
 
     from functools import partial
 
+    from seals import base_envs as envs
```

Comment:
I tend to prefer avoiding renaming imports (one less name to track) unless it's an established convention (e.g. `numpy as np`) or would be overly long otherwise. Does that or another reason apply here?

Reply:
Changed it.